README.md

```diff
@@ -11,8 +11,6 @@ Turanic is a fork of GenisysPro
 ## Can I test this before I download it?
 Yes, you can. There are our member who already use it, and at times there are dev servers running and you might be lucky enough to get in..<br>
 
-IP: **play.networkgames.eu**    
-Port: **19132**
```

Comment:
certainly, I still can not wait, my players complain about not updating the server. yesterday I try to update the server with Turanic and every 30 minutes it was offline

Reply:
I send the error on discord, the answer of @Enes5519 Does not do anything, good response